code/datums/status_effects/neutral.dm

```diff
@@ -342,3 +342,35 @@
 	tick_interval = INFINITY
 	status_type = STATUS_EFFECT_REFRESH
 	alert_type = null
+
+/*
+* Freezes the player in a block of ice
+* Will be removed when the required reagent is removed too
+*/
+/obj/screen/alert/status_effect/freon/stasis/cryosenium
+	desc = "You're frozen inside of a protective ice cube! While inside, you can't do anything, but are immune to harm! You will be free when the chem runs out."
+
+/datum/status_effect/cryosenium
+	id = "cryosenium"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = 600 //Will remove self when reagent is processed, though limited to 60s to be safe. I suppose if you're freezing the dying you want time but I'd rather have this incase oops 1000u
+	alert_type = /obj/screen/alert/status_effect/freon/stasis/cryosenium
+	var/obj/structure/ice_stasis/cube
+
+/datum/status_effect/cryosenium/on_apply()
+	cube = new /obj/structure/ice_stasis(get_turf(owner))
+	cube.color = "#03dbfc"
+	cube.anchored = 1
+	owner.forceMove(cube)
+	owner.status_flags |= GODMODE
```

Comment:
hmmm

Reply:
oh no no no

Reply:
It partially borrows from the other ice cube effect, is there an alternative that would be better?

Reply:
to do what

Reply:
It's alright, I'm converting it to stasis instead.

Reply:
Okay it's stasis now